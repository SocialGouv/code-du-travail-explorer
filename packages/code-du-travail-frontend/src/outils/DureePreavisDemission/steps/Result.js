import React from "react";
import PropTypes from "prop-types";
import { Toast } from "@cdt/ui";

import { SectionTitle } from "../../common/stepStyles";
import { filterSituations, recapSituation } from "./situation";

function StepResult({ form }) {
  const { values } = form.getState();
  const possibleSituations = filterSituations(values);
  switch (possibleSituations.length) {
    case 1: {
      const [situation] = possibleSituations;
      const { idcc, label: ccLabel } = situation.criteria.branche;
      return (
        <>
          <SectionTitle>Durée du préavis</SectionTitle>
          <p>
            En cas de démission, la convention collective{" "}
            <strong>
              {ccLabel} ({idcc})
            </strong>{" "}
            prévoit le respect d’un préavis d’une durée de{" "}
            <strong>{situation.answerCc}</strong> pour un salarié{" "}
            {recapSituation(situation.criteria)}
          </p>
          {situation.refCc && situation.refCcUrl && (
            <p>
              <a
                href={situation.refCcUrl}
                title="Consultez l'article de votre convetion collective"
              >
                {situation.refCc}
              </a>
            </p>
          )}
          <Toast variant="info">
            Le contrat de travail, un accord collectif d’entreprise ou un usage
            peuvent prévoir une durée de préavis plus courte. Vous pouvez
            également convenir d’une dispense par accord avec l’employeur.
          </Toast>
        </>
      );
    }
    default:
      return (
        <>
          <p>
            Le code du travail ne prévoit pas une durée précise du préavis de
            démission. Il prévoit qu’une convention collective ou un accord
            d’entreprise, voire un usage, en prévoit les durées et modalités.
          </p>
        </>
      );
  }
}

StepResult.propTypes = {
  form: PropTypes.shape({
    getState: PropTypes.func.isRequired
  })
};

export { StepResult };
