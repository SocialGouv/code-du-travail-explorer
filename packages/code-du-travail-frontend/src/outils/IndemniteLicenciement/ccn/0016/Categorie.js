import React from "react";
import PropTypes from "prop-types";
import { Field } from "react-final-form";
import { Select } from "@socialgouv/react-ui";

import { required } from "../../../common/validators";
import { Label, SectionTitle } from "../../../common/stepStyles";

export const OUVRIER = "ouvrier";
export const TAM = "tam";
export const CADRE = "cadre";

export const Categories = {
  [OUVRIER]: "Ouvriers et employés",
  [TAM]: "Techniciens et agents de maîtrise (TAM)",
  [CADRE]: "Cadre"
};

function Categorie({ name }) {
  return (
    <>
      <SectionTitle>Quelle est la catégorie du salarié&nbsp;?</SectionTitle>
      <p>Cette information se trouve sur le bulletin de salaire.</p>
      <Field name={name} validate={required}>
        {({ input }) => (
          <>
            <Label htmlFor={`input-${name}`}>Sélectionnez une catégorie</Label>
            <Select {...input} id={`input-${name}`}>
              <option disabled value="" />
              {Object.entries(Categories).map(([value, label]) => (
                <option key={value} value={value}>
                  {label}
                </option>
              ))}
            </Select>
          </>
        )}
      </Field>
    </>
  );
}

Categorie.propTypes = {
  name: PropTypes.string.isRequired
};

export { Categorie };
