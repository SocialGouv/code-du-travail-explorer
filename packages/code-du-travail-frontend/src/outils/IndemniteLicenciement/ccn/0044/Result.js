import React from "react";
import {
  getIndemnite as getIndemniteConventionnelle,
  getSalaireRef as getSalaireRefConventionnel
} from "./indemnite";
import { getIndemniteFromFinalForm } from "../../indemnite";
import { IndemniteCCn } from "../../components/IndemniteConventionnelle";

function Result({ form }) {
  const state = form.getState();

  const {
    salaires,
    salaire,
    anciennete,
    dateNotification,
    isEco,
    hasOpe,
    age,
    groupe,
    branche
  } = state.values;

  const {
    salaireRefLegal,
    indemniteLegale,
    formuleLegale,
    inputLegals
  } = getIndemniteFromFinalForm(form);

  const salaireRef = getSalaireRefConventionnel({
    salaireRefLegal,
    groupe,
    salaires,
    salaire,
    dateNotification
  });
  const {
    indemniteConventionnelle,
    formuleConventionnelle,
    inputConventionnels
  } = getIndemniteConventionnelle({
    salaireRef,
    indemnite: indemniteLegale,
    anciennete,
    isEco,
    hasOpe,
    age,
    groupe
  });

  return (
    <IndemniteCCn
      indemniteConventionnelle={indemniteConventionnelle}
      indemniteLegale={indemniteLegale}
      formuleConventionnelle={formuleConventionnelle}
      formuleLegale={formuleLegale}
      branche={branche}
      inputLegals={inputLegals}
      inputConventionnels={inputConventionnels}
    />
  );
}

export { Result };
