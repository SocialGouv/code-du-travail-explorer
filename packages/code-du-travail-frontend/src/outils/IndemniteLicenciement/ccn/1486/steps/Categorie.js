import React from "react";
import { Field } from "react-final-form";
import { OnChange } from "react-final-form-listeners";

import { SectionTitle, Label } from "../../../../common/stepStyles";
import { required } from "../../../../common/validators";
import { stepAnciennete, stepSalaire } from "../index";

export const CATEGORIE_KEY = "brancheCategorie";

const CATEGORIES = {
  ETAM: "Employé, Technicien, Agent de Maîtrise",
  IC: "Ingénieur Cadre",
  CENI: "Chargé d'enquête non intermittent",
  CEI: "Chargé d'enquête intermittent"
};

export const Categorie = ({ dispatch }) => (
  <>
    <SectionTitle>Catégorie professionnelle</SectionTitle>
    <p>
      Quelle est votre catégorie d’après la convention collective ?<br />
      Vous pouvez la trouver sur votre bulletin de paie.
    </p>
    <Field name={CATEGORIE_KEY} validate={required}>
      {({ input }) => (
        <>
          <Label htmlFor="categorie">
            Sélectionnez une catégorie professionnelle
          </Label>
          <select {...input} id="categorie">
            <option disabled value="">
              Sélectionner
            </option>
            {Object.entries(CATEGORIES).map(([key, label]) => (
              <option key={key} value={key}>
                {label}
              </option>
            ))}
          </select>
        </>
      )}
    </Field>
    <OnChange name={CATEGORIE_KEY}>
      {currentCategorie => {
        if (currentCategorie === "CEI") {
          dispatch({ type: "remove_step", payload: stepSalaire.name });
        } else {
          dispatch({
            type: "add_step",
            payload: { insertAfter: stepAnciennete.name, step: stepSalaire }
          });
        }
      }}
    </OnChange>
  </>
);
