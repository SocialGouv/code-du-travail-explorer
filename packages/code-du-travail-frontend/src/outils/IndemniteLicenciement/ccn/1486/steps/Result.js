import React from "react";
import styled from "styled-components";
import { Toast, theme } from "@cdt/ui-old";

import { getIndemniteConventionnelle } from "../indemnite";
import { getIndemniteFromFinalForm } from "../../../indemnite";
import { round } from "../../../../common/math";
import { Highlight } from "../../../../common/stepStyles";
import { IndemniteCCn } from "../../../components/IndemniteConventionnelle";

function Result({ form }) {
  const data = form.getState().values;

  const { indemniteLegale, infoCalculLegal } = getIndemniteFromFinalForm(form);

  const {
    indemniteConventionnelle,
    infoCalculConventionnel,
    error
  } = getIndemniteConventionnelle(data);

  const IndemniteWarning = () => (
    <StyledToast variant="warning">
      Si un emploi équivalent a été trouvé
      <ul>
        <li>par votre employeur</li>
        <li>dans une autre entreprise</li>
        <li>avant la fin du préavis</li>
        <li>et que vous l’acceptez</li>
      </ul>
      Alors l’indemnité de licenciement conventionnelle est réduite d’un tiers
      et passe donc à{" "}
      <Highlight>{round((indemniteConventionnelle * 2) / 3)}&nbsp;€</Highlight>
      <br />
      Attention, ce tiers ({round(indemniteConventionnelle / 3)}&nbsp;€) vous
      est dû si la période d’essai dans ce nouvel emploi reste sans suite.
    </StyledToast>
  );

  return (
    <IndemniteCCn
      indemniteConventionnelle={indemniteConventionnelle}
      indemniteLegale={indemniteLegale}
      infoCalculLegal={infoCalculLegal}
      infoCalculConventionnel={infoCalculConventionnel}
      branche={data.branche}
      error={error}
    >
      {indemniteConventionnelle > indemniteLegale ? <IndemniteWarning /> : null}
    </IndemniteCCn>
  );
}

export { Result };

const { spacing } = theme;

const StyledToast = styled(Toast)`
  margin: ${spacing.interComponent} 0;
`;
