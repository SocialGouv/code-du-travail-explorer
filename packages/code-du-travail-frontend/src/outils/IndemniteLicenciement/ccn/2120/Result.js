import React from "react";
import { getIndemniteFromFinalForm } from "../../indemnite";

import { IndemniteCCn } from "../../components/IndemniteConventionnelle";
import { getIndemnite, getSalaireRef } from "./indemnite";

export function Result({ form }) {
  const state = form.getState();

  const {
    hasTempsPartiel = false,
    hasSameSalaire = false,
    salairePeriods = [],
    salaires = [],
    salaire,
    anciennete,
    dateEntree,
    dateSortie,
    branche,
    motif,
    categorie,
  } = state.values;

  const { indemniteLegale, infoCalculLegal } = getIndemniteFromFinalForm(form);
  const salaireRef = getSalaireRef({
    hasTempsPartiel,
    hasSameSalaire,
    salaire,
    salairePeriods,
    salaires,
    anciennete,
  });
  const {
    error,
    indemniteConventionnelle,
    infoCalculConventionnel,
  } = getIndemnite({
    salaireRef,
    dateEntree,
    dateSortie,
    anciennete,
    categorie,
    motif,
  });
  return (
    <IndemniteCCn
      indemniteConventionnelle={indemniteConventionnelle}
      indemniteLegale={indemniteLegale}
      infoCalculLegal={infoCalculLegal}
      infoCalculConventionnel={infoCalculConventionnel}
      branche={branche}
      error={error}
    />
  );
}
