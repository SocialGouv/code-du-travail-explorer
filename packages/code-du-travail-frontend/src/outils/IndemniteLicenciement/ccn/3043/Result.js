import React from "react";

import {
  getIndemnite as getIndemniteConventionnelle,
  getSalaireRef as getSalaireRefConventionnel
} from "./indemnite";

import { getIndemniteFromFinalForm } from "../../indemnite";
import { IndemniteCCn } from "../../components/IndemniteConventionnelle";

function ResultProprete({ form }) {
  const state = form.getState();

  const {
    salaires = [],
    salaire,
    anciennete,
    dateNotification,
    isEco,
    hasOpe,
    age,
    groupe,
    branche
  } = state.values;

  const {
    salaireRefLegal,
    indemniteLegale,
    infosCalculLegal
  } = getIndemniteFromFinalForm(form);

  const salaireRef = getSalaireRefConventionnel({
    salaireRefLegal,
    groupe,
    salaires,
    salaire,
    dateNotification
  });

  const {
    indemniteConventionnelle,
    infosCalculConventionnel,
    error
  } = getIndemniteConventionnelle({
    salaireRef,
    indemnite: indemniteLegale,
    anciennete,
    isEco,
    hasOpe,
    age,
    groupe
  });

  return (
    <IndemniteCCn
      indemniteConventionnelle={indemniteConventionnelle}
      indemniteLegale={indemniteLegale}
      infosCalculLegal={infosCalculLegal}
      infosCalculConventionnel={infosCalculConventionnel}
      branche={branche}
      error={error}
    />
  );
}

export default ResultProprete;
