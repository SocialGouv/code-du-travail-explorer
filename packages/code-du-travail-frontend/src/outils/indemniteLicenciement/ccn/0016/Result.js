import React from "react";
import MathJax from "react-mathjax-preview";
import { Container, Alert, Button } from "@cdt/ui";
import { getIndemnite, getSalaireRef } from "../../indemnite";
import { branches } from "../../branches";
import { SectionTitle, Highlight, SmallText } from "../../stepStyles";
import {
  getIndemnite as getIndemniteConventionnelle,
  getSalaireRef as getSalaireRefConventionnel
} from "./indemnite";

import { ErrorBoundary } from "../../../../common/ErrorBoundary";

export function Result({ form }) {
  const state = form.getState();

  const {
    hasTempsPartiel = false,
    hasSameSalaire = false,
    inaptitude = false,
    salairePeriods = [],
    salaires = [],
    primes = [],
    salaire,
    anciennete,
    dateNotification,
    branche,
    age,
    categorie,
    tamDuration,
    cadreDuration,
    hasRetirementAge = false
  } = state.values;

  const selectedBranche = branches.find(br => br.value === branche);

  const salaireRef = getSalaireRef({
    hasTempsPartiel,
    hasSameSalaire,
    salaire,
    salairePeriods,
    salaires,
    anciennete,
    primes
  });

  const { indemnite } = getIndemnite({
    salaireRef,
    anciennete,
    inaptitude,
    dateNotification
  });

  const salaireRefConventionnel = getSalaireRefConventionnel({
    hasTempsPartiel,
    hasSameSalaire,
    salaire,
    salairePeriods,
    salaires,
    primes,
    anciennete,
    categorie
  });

  const {
    error,
    indemniteConventionnelle,
    formula
  } = getIndemniteConventionnelle({
    age,
    categorie,
    salaireRef: salaireRefConventionnel,
    indemnite,
    anciennete,
    tamDuration,
    cadreDuration,
    hasRetirementAge
  });

  return (
    <Container>
      <SectionTitle>{selectedBranche.label}</SectionTitle>
      {error ? (
        <Alert>
          Aucune indemnité de licenciement n’est prévue en deça de 2 ans
          d’ancienneté.
        </Alert>
      ) : (
        <IndemniteConventionnelle
          montant={indemniteConventionnelle}
          indemniteLegale={indemnite}
          formula={formula}
        />
      )}
    </Container>
  );
}

function IndemniteConventionnelle({ montant, formula, indemniteLegale }) {
  return (
    <React.Fragment>
      <p>
        Le montant de l’indemnité est <Highlight>{montant} €</Highlight>{" "}
        <SmallText>
          {montant > indemniteLegale
            ? "sur la base du calcul de l'indemnité conventionelle"
            : "sur la base du calcul de l’indemnité légale"}
        </SmallText>
      </p>
      <br />
      <details>
        <summary>Voir le detail du calcul</summary>
        <ErrorBoundary>
          <MathJax math={"`" + formula + "`"} />
        </ErrorBoundary>
      </details>
      <br />
      <br />
      <Button> Recommencer une simulation </Button>
    </React.Fragment>
  );
}
