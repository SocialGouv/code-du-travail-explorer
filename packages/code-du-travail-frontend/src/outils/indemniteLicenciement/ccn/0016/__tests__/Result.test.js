import React from "react";
import { render } from "react-testing-library";
import { Result } from "../Result";
import { Form } from "react-final-form";
import { CADRE } from "../Categorie";

const initialValues = {
  contrat: "cdi",
  fauteGrave: false,
  inaptitude: false,
  dateEntree: "2009-01-06",
  anciennete: 10.58,
  dateNotification: "2019-06-03",
  dateSortie: "2019-09-02",
  hasAbsenceProlonge: false,
  absencePeriods: [],
  hasTempsPartiel: false,
  salairePeriods: [],
  hasSameSalaire: true,
  salaires: [],
  salaire: "2000",
  branche: "0016",
  categorie: CADRE,
  tamDuration: "5",
  cadreDuration: "120",
  age: "64",
  hasRetirementAge: true
};
describe("<Result />", () => {
  it("should render", () => {
    const onSubmit = jest.fn();
    const { container } = render(
      <Form onSubmit={onSubmit} initialValues={initialValues}>
        {({ form }) => <Result form={form} />}
      </Form>
    );
    expect(container).toMatchSnapshot();
  });
});
