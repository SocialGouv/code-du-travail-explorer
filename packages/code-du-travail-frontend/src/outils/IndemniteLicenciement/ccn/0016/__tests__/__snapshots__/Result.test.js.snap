// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Result /> should render 1`] = `
.c21 {
  display: inline-block;
  padding: 0.625rem 1.5rem;
  font-weight: 600;
  font-size: 1rem;
  line-height: inherit;
  text-align: center;
  vertical-align: middle;
  border: 1px solid;
  border-radius: 1.5rem;
  cursor: pointer;
  -webkit-transition: background-color 250ms ease;
  transition: background-color 250ms ease;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  color: #fff;
  background: #006be6;
  border-color: #006be6;
}

.c21:not([disabled]):hover,
.c21:not([disabled]):active,
.c21:not([disabled]):focus {
  color: #fff;
  background: #07f;
}

.c21[disabled] {
  color: rgba(255,255,255,0.4);
  cursor: not-allowed;
}

.c17 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  box-sizing: border-box;
  width: auto;
  min-height: 48px;
  color: #434956;
  background-color: #fff;
  border-color: #e5f1fe;
  border-style: solid;
  border-width: 1px;
  border-radius: 0.25rem;
  box-shadow: none;
  -webkit-animation: none;
  animation: none;
}

.c19 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 0;
  -webkit-flex-grow: 0;
  -ms-flex-positive: 0;
  flex-grow: 0;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 1rem;
  background-color: #e5f1fe;
}

.c20 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  padding: 1rem;
  color: #434956;
  text-align: left;
}

.c0 {
  margin-top: 2rem;
  margin-bottom: 1.25rem;
  font-size: 1.625rem;
}

.c1 {
  color: #005994;
  font-weight: 700;
  font-size: 1.375rem;
}

.c9 {
  margin-bottom: 1rem;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c6 {
  position: relative;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: calc(50% * 0.9533911504424778);
  padding: 1em;
  background-color: #4c5467;
  border-radius: 0.25rem;
}

.c15 {
  position: relative;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: calc(50% * 1);
  padding: 1em;
  border-radius: 0.25rem;
}

.c7 {
  margin-left: 1rem;
  color: #4c5467;
  font-size: 1.375rem;
}

.c16 {
  margin-left: 1rem;
  font-size: 1.875rem;
}

.c8 {
  margin-top: 1.25rem;
}

.c10 {
  font-weight: bold;
  font-size: 0.875rem;
}

.c11 {
  list-style-type: disc;
}

.c12 {
  font-size: 0.875rem;
}

.c13 {
  margin: 1rem 0;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c3 {
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
  padding: 0 0.625rem;
}

.c14 {
  -webkit-order: 0;
  -ms-flex-order: 0;
  order: 0;
  padding: 0 0.625rem;
}

.c4 {
  font-size: 1.375rem;
}

.c18 {
  margin: 1.25rem 0;
}

@media (max-width:980px) {
  .c5 {
    -webkit-flex-flow: wrap;
    -ms-flex-flow: wrap;
    flex-flow: wrap;
  }
}

@media (max-width:980px) {
  .c6 {
    width: calc(100% * 0.9533911504424778);
  }
}

@media (max-width:980px) {
  .c15 {
    width: calc(100% * 1);
  }
}

@media (max-width:980px) {
  .c7 {
    width: 100%;
    margin-left: 0;
  }
}

@media (max-width:980px) {
  .c16 {
    width: 100%;
    margin-left: 0;
  }
}

<div>
  <h2
    class="c0"
  >
    Convention collective nationale des transports routiers et activités auxiliaires du transport du 21 décembre 1950
  </h2>
  <p>
    À partir des éléments que vous avez saisis, votre indémnité de licenciement est estimée à
     
    <strong
      class="c1"
    >
      5,650.00
       € brut
    </strong>
  </p>
  <p>
    Il s’agit du montant le plus favorable entre votre indemnité légale et votre indemnité conventionnelle.
  </p>
  <div
    class="c2"
  >
    <section
      class="c3"
    >
      <h2
        class="c4"
      >
        Votre indemnite légale
      </h2>
      <div
        class="c5"
      >
        <div
          class="c6"
        />
        <span
          class="c7"
        >
          5,386.66
           € brut
        </span>
      </div>
      <details
        class="c8"
      >
        <summary
          class="c9"
        >
          Voir le detail du calcul
        </summary>
        <strong
          class="c10"
        >
          Élements saisis :
        </strong>
        <ul
          class="c11"
        >
          <li
            class="c12"
          >
            Salaire de réference (Sref)
             : 
            2000
          </li>
          <li
            class="c12"
          >
            Licenciement pour inaptitude
             : 
            non
          </li>
          <li
            class="c12"
          >
            Ancienneté totale (A)
             : 
            10.58
          </li>
          <li
            class="c12"
          >
            Ancienneté au delà de 10ans (A2)
             : 
            0.58
          </li>
        </ul>
        <strong
          class="c10"
        >
          Formule
        </strong>
        <div
          class="c13"
        >
          <div
            id="react-mathjax-preview"
          >
            <div
              id="react-mathjax-preview-result"
            >
              \`(1/4 * Sref * 10) + (1/3 * Sref * "A2")\`
            </div>
          </div>
        </div>
      </details>
    </section>
    <section
      class="c14"
    >
      <h2
        class="c4"
      >
        Votre indemnite conventionnelle
      </h2>
      <div
        class="c5"
      >
        <div
          class="c15"
        />
        <span
          class="c16"
        >
          5,650.00
           € brut
        </span>
      </div>
      <details
        class="c8"
      >
        <summary
          class="c9"
        >
          Voir le detail du calcul
        </summary>
        <strong
          class="c10"
        >
          Élements saisis :
        </strong>
        <ul
          class="c11"
        >
          <li
            class="c12"
          >
            salaire de référence (Sref)
             : 
            2000
          </li>
          <li
            class="c12"
          >
            catégorie
             : 
            cadre
          </li>
          <li
            class="c12"
          >
            durée comme cadre, en mois, (Dc)
             : 
            120
          </li>
          <li
            class="c12"
          >
            durée comme TAM, en mois, (Dt)
             : 
            5
          </li>
          <li
            class="c12"
          >
            en age de partir à la retraite
             : 
            oui
          </li>
          <li
            class="c12"
          >
            majoration
             : 
            4000
          </li>
          <li
            class="c12"
          >
            minoration
             : 
            6600
          </li>
        </ul>
        <strong
          class="c10"
        >
          Formule
        </strong>
        <div
          class="c13"
        >
          <div
            id="react-mathjax-preview"
          >
            <div
              id="react-mathjax-preview-result"
            >
              \`4/10 * Sref * Dc + 3/10 * Sref * Dt  + "majoration" - "minoration"\`
            </div>
          </div>
        </div>
      </details>
    </section>
  </div>
  <div
    class="c17 c18"
  >
    <div
      class="c19"
    >
      <svg
        fill="none"
        height="24"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <circle
          cx="12"
          cy="12"
          r="10"
        />
        <line
          x1="12"
          x2="12"
          y1="16"
          y2="12"
        />
        <line
          x1="12"
          x2="12"
          y1="8"
          y2="8"
        />
      </svg>
    </div>
    <div
      class="c20"
      role="alert"
    >
      Un accord collectif d’entreprise, le contrat de travail et un usage peuvent prévoir une formule de calcul plus avantageuse pour le salarié. Dans ce cas, le salarié perçoit l’indemnité la plus élevée.
    </div>
  </div>
  <button
    class="c21"
  >
    Recommencer une simulation
  </button>
</div>
`;
